### PR TITLE
Simplfy repeated information in the docs

### PR DESCRIPTION
**Pull request by smyrick**
### :pencil: Description
Refactor and reorganize the documentation so that we don't have duplication information across pages. This makes it easier to update on future changes.

Also add the recent blog posts to the list.

### :link: Related Issues
N\A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
 
-GraphQL Kotlin is a collection of libraries built on top of [graphql-java](https://www.graphql-java.com/) that aim to simplify running a GraphQL server in Kotlin
+GraphQL Kotlin is a collection of libraries, built on top of [graphql-java](https://www.graphql-java.com/), that aim to simplify running GraphQL in Kotlin
```
Comment (dariuszkuc) [inline README.md:8]: maybe we should expand on "running GraphQL in Kotlin" ?

Comment (smyrick) [reply README.md:8]: That is just the tag line. I wanted to keep it fairly short. I have updated the main page the include links to specific parts

* Running server
* Creating client
* Generating schema

File: docs/examples.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ id: examples
 title: Examples
 ---
 
+A collection of example apps that use graphql-kotlin libraries to test and demonstrate usages can be found in the [examples module](https://github.com/ExpediaGroup/graphql-kotlin/tree/master/examples).
```
Comment (dariuszkuc) [inline docs/examples.md:6]: can we add a section for new client examples?

File: docs/getting-started.md [hunk 3]
```diff
@@ -30,96 +32,38 @@ Additional resources
...
+## Creating a Client
+`graphql-kotlin-plugins` can be used to generate a `graphql-kotlin-client` from an existing schema that is easy to use and type-safe.
 
-### Class `TopLevelObject`
+See the docs in [Client Overview](./client/client-overview.md).
 
-`toSchema` uses Kotlin reflection to build a GraphQL schema from given classes using `graphql-java`'s schema builder. We
-don't just pass a `KClass` though, we have to actually pass an object, because the functions on the object are
-transformed into the data fetchers. In most cases, a `TopLevelObject` can be constructed with just an object:
+## Generating a Schema
```
Comment (dariuszkuc) [inline docs/getting-started.md:65]: I think we should move this up -> i.e. start with generating a schema, then spring server (as it references the schema generation) and close with the client (which runs against a schema).

File: docs/schema-generator/schema-generator-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  id: Int!
+  value: String!
+}
+```
+
+Any `public` functions defined on a query, mutation, or subscription Kotlin class will be translated into GraphQL fields on the object
+type. `toSchema` will then recursively apply Kotlin reflection on the specified classes to generate all
+remaining object types, their properties, functions, and function arguments.
+
+The `GraphQLSchema` generated can then be used to expose a GraphQL API endpoint.
```
Comment (dariuszkuc) [inline docs/schema-generator/schema-generator-getting-started.md:58]:
```suggestion
The generated `GraphQLSchema` can then be used to expose a GraphQL API endpoint.
```

File: website/package-lock.json [hunk 5]
```diff
@@ -1864,9 +1864,9 @@
       }
     },
     "buffer": {
-      "version": "5.4.3",
-      "resolved": "https://artylab.expedia.biz/api/npm/public-npm-virtual/buffer/-/buffer-5.4.3.tgz",
-      "integrity": "sha1-P7ycaetxPTI+P8Gole7gcQwHIRU=",
+      "version": "5.6.0",
+      "resolved": "https://artylab.expedia.biz/api/npm/public-npm-virtual/buffer/-/buffer-5.6.0.tgz",
```
Comment (dariuszkuc) [inline website/package-lock.json:1868]: can you regenerate it using public npmjs registry?

File: .npmrc [hunk 6]
```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```
Comment (dariuszkuc) [inline .npmrc:1]: should this file be placed under federated gateway and website directories instead (same as .nvmrc)?

Comment (smyrick) [reply .npmrc:1]: Sure, I can move it

**Pull request merged**